app/(client)/profile/layout.tsx

```diff
@@ -0,0 +1,214 @@
+'use client';
+import { useCallback, useEffect, useState } from 'react';
+import { AxiosError } from 'axios';
+import { KeyRound, Layers, User } from 'lucide-react';
+import Link from 'next/link';
+
+import { BreadcrumbItemType, Breadcrumbs } from '@/components/ui/breadcrumb';
+import { Button } from '@/components/ui/button';
+import {
+  Dialog,
+  DialogContent,
+  DialogFooter,
+  DialogTitle,
+  DialogTrigger,
+} from '@/components/ui/dialog';
+import { toast } from '@/components/ui/toast/use-toast';
+import useAuth from '@/hooks/useAuth';
+import { api } from '@/lib/api';
+import { useProfileStore } from '@/store/profile-store';
+import { UserI } from '@/types/auth/user.interface';
+
+const BREADCRUMBS_ITEMS: BreadcrumbItemType[] = [
+  {
+    icon: Layers,
+    href: '/',
+    label: 'Головна',
+  },
+  {
+    icon: User,
+    href: '/profile',
+    label: 'Персональний аккаунт',
+  },
+];
+
+export default function ClientLayout({
+  children,
+}: Readonly<{
+  children: React.ReactNode;
+}>) {
+  const [dialogDeleteAccountOpen, setDialogDeleteAccountOpen] = useState(false);
+  const [errors, setErrors] = useState(null);
+  const { userData, setUserData, isProfileMenuActive, setIsProfileMenuActive } =
+    useProfileStore(state => state);
+  const { loggedIn, logout } = useAuth();
+
+  const fetchUser = useCallback(async () => {
+    try {
+      const res = await api.get<UserI>('/user');
+      setUserData(res.data);
+      setErrors(null);
+    } catch (error) {
+      if (error instanceof AxiosError) {
+        setErrors(error.response?.data.message || 'Сталася помилка');
+      }
+    }
+  }, [setUserData]);
+
+  useEffect(() => {
+    if (loggedIn) {
+      fetchUser();
+    }
+  }, [loggedIn, setUserData, fetchUser]);
+
+  const handleDeleteUser = async () => {
+    try {
+      await api.delete('/auth/user').then(() => {
```

Comment:
You don't need to use both await and .then() in the same code block because they serve the same purpose